receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -189,8 +190,13 @@ func (rw *receiverWorker) export(longLivedCtx context.Context, tracedata *consum
 		return
 	}
 
+	ctx := context.Background()
+	if c, ok := client.FromGRPC(longLivedCtx); ok {
+		ctx = client.NewContext(ctx, c)
```

Comment:
Why can't the long live ctx be used as an input here?

Reply:
longLivedCtx is bound to the stream handler loop and remains active for the lifetime of the connection. The new context is used per request (incoming batch) to trace handling of each batch as a single independent request. This is not a new change and continues to work as before.